services/parser-database/app.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from flask import Flask  # pylint: disable=import-error
+from flask import request  # pylint: disable=import-error
+from flask import jsonify  # pylint: disable=import-error
+
+from parser import parse_all_documents  # pylint: disable=import-error
+from parser import get_articles_that_match_keywords  # pylint: disable=import-error
+from parser import get_article_by_number  # pylint: disable=import-error
+
+app = Flask(__name__)
+
+
+@app.route('/parse', endpoint='parser', methods=['POST'])
+def trigger_parsing():
+    # return parser.parse_without_database()
+    try:
+        parse_all_documents()
+    except Exception as e:
+        print(e)
+        return "Internal Server Error", 500
+    return "Sucessful Operation", 200
```

Comment:
nit: What happens when this gets called multiple times concurrently? The API spec says it should return `409` but right now it looks like it will just try to parse in parallel for every request.

(I don't think this needs to be addressed in this PR, but it should probably be marked as a TODO and linked to an issue for Post-MVP)

Reply:
Yes, if I recall a conversation we had, that was post-MVP. I'll go ahead and create a TODO for that.